examples/trials/mnist-tfv1/config_heterogeneous.yml

```diff
@@ -3,7 +3,7 @@ experimentName: example_mnist
 trialConcurrency: 3
 maxExecDuration: 1h
 maxTrialNum: 10
-trainingServicePlatform: heterogeneous
+trainingServicePlatform: hybrid
```

Comment:
`config_heterogeneous.yml ` -> `config_hybrid.yml `

Reply:
let me fix this one in my pr :)